perfkitbenchmarker/providers/ibmcloud/ibmcloud_disk.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing classes related to IBM Cloud disks """
+
+import json
+import logging
+import time
+import re
+
+from absl import flags
+from perfkitbenchmarker import disk
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import vm_util
+
+from perfkitbenchmarker.providers.ibmcloud import ibm_api as ibm
+
+FLAGS = flags.FLAGS
+
+ATTACHED = 'attached'
+AVAILABLE = 'available'
+DEFAULT_DISK_SIZE = 1000
+
+
+class IbmCloudDisk(disk.BaseDisk):
+  """Object representing a IBM Cloud Disk"""
+
+  def __init__(self, disk_spec, name, zone):
+    super(IbmCloudDisk, self).__init__(disk_spec)
+    self.name = name
+    self.zone = zone
+    self.vol_id = None
+    self.attached_vm = None
+    self.attached_vdisk_URI = None
+    self.device_path = None
+    self.data_disk_size = FLAGS.data_disk_size if FLAGS.data_disk_size is not None else DEFAULT_DISK_SIZE
+
+  def _Create(self):
+    """Creates an external block volume."""
+    volcmd = ibm.IbmAPICommand(self)
+    volcmd.flags['name'] = self.name
```

Comment:
Nit: I find it easier to read
  volcmd.flags.update(
{'name': self.name,
'zone' :self.zone,
....
'capacity' : self.data_disk_size,
}

Reply:
yes it is easier to read, updated.